### ISSUE TITLE
Expand outline logic doesn't deal with TabContainers properly

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#1913]
- New HTML File (hi-fi page)
- Add TabContainer and then add Button into first tab pane

Result=> Console shows this message:

`Could not expand path at TabContainer_0`

**Comment by peller** [comment]
possible dup of #1909

**Comment by JonFerraiolo** [comment]
I believe I found the bug. I have a fix but I'm not sure about the side effects. The bug is in DijitWidget.js:77. For some reason we are putting _dvWidget onto dijitWidget.containerNode. For widgets where containerNode!=domNode, then we are putting the same _dvWidget pointer onto two different nodes. On my system I have commented out that assignment and now I'll do a bunch of testing to see what breaks.
